R/respiratory-variables.R

```diff
@@ -0,0 +1,80 @@
+#' Respiratory condition variable
+#' 
+#' This function creates a derived variable that determines if a subject has a respiratory condition
+#' 
+#' This function is for CCHS cycles that only use COPD and Emphysema as a combined variable
```

Comment:
Can you provide more details? Which survey years us only COPD and emphysema? What are the variable names and categories?

Reply:
Can you describe the use case? e.g. 'use this variable if you want to harmonize....'

Reply:
The title of the different respiratory variables is the same. Could you differentiate in the title?
e.g.
"Respiratory variable - combed COPD and emphysema"
"Respiratory variable - combed COPD, emphysema and bronchitis"

Reply:
All 3 of these functions feed into the same derived variable. The reason 3 functions had to be done is to account for the fact that different respiratory variables used between CCHS cycles. If one function was to be used that included all the resp variables, there would be errors in RecWTable() when trying to transform a CCHS cycle that is missing one or more of the resp variables. I will explain further in the .R file

Reply:
Using different tittles might be misleading since it is the same variable. And he does differentiate in the description of the function how that variable was calculated ie: combed COPD and emphysema, combed COPD, emphysema and bronchitis.

Reply:
OK. I think that I just need to be walked through. If it is unclear to me it probably will be unclear to others.